test/connectors/PenaltiesConnectorSpec.scala

```diff
@@ -153,10 +153,21 @@ class PenaltiesConnectorSpec extends SpecBase {
       result.get.toString() shouldBe appealDataAsJson.toString()
     }
 
-    s"return $Some $JsValue when the connector call succeeds for LPP" in new Setup {
+s"return $Some $JsValue when the connector call succeeds for LPP" in new Setup {
       when(mockHttpClient.GET[HttpResponse](Matchers.any(), Matchers.any(), Matchers.any())(Matchers.any(), Matchers.any(), Matchers.any()))
         .thenReturn(Future.successful(HttpResponse(Status.OK, appealDataAsJsonLPP.toString())))
-      when(mockAppConfig.appealLPPDataForPenaltyAndEnrolmentKey(Matchers.any(), Matchers.any(), Matchers.any()))
+      when(mockAppConfig.appealLPPDataForPenaltyAndEnrolmentKey(Matchers.any(), Matchers.any(),Matchers.any()))
+        .thenReturn("http://url/url")
+
+      val result = await(connector.getAppealsDataForPenalty("12345", "123456789", isLPP = true, isAdditional = false))
+      result.isDefined shouldBe true
+      result.get.toString() shouldBe appealDataAsJsonLPP.toString()
+    }
+
+    s"return $Some $JsValue when the connector call succeeds for LPP additional" in new Setup {
```

Comment:
Seems this test is a duplicate, it also sets `isAdditional` to false, maybe it can be removed?
https://github.com/hmrc/penalties-appeals-frontend/blob/aac739ba77888803de21abb62e4043bb886956bf/test/connectors/PenaltiesConnectorSpec.scala#L167-L187